aws.tf

```diff
@@ -7,6 +7,12 @@ terraform {
   }
 }
 
+# Default
+provider "aws" {
+  version = "2.16.0"
+  region  = "us-east-1"
```

Comment:
No alias?

Reply:
Yes, no alias means it's the default provider, thus the comment above. Unfortunately there is no more explicit options (e.g. `default = true` property) AFAIK

Reply:
This is kind of confusing feature of terraform. One of the providers can be without an alias and it will be used for all resources that does not implicitly specify a provider (like certificates https://github.com/aeternity/terraform-aws-testnet-api/pull/6/files#diff-208348eb365bcea2f79ae7be695303aeR1). If all have aliases, there will be no default then such resources will fail. In addition Certificates can be only created in us-east-1 region, however they still require to have a provider, either implicitly defined or use a default which is set to `us-east-1`